src/SharedWeb/Utilities/ServiceCollectionExtensions.cs

```diff
@@ -540,10 +540,14 @@ public static void UseForwardedHeaders(this IApplicationBuilder app, IGlobalSett
         if (!globalSettings.UnifiedDeployment)
         {
             // Trust the X-Forwarded-Host header of the nginx docker container
-            var nginxIp = Dns.GetHostEntry("nginx").AddressList.FirstOrDefault();
-            if (nginxIp != null)
-            {
-                options.KnownProxies.Add(nginxIp);
+            try {
+                var nginxIp = Dns.GetHostEntry("nginx")?.AddressList.FirstOrDefault();
+                if (nginxIp != null)
+                {
+                    options.KnownProxies.Add(nginxIp);
+                }
+            } catch {
+                // Ignore DNS errors
```

Comment:
C# style bracing please 😄 

```suggestion
            try
            {
                var nginxIp = Dns.GetHostEntry("nginx")?.AddressList.FirstOrDefault();
                if (nginxIp != null)
                {
                    options.KnownProxies.Add(nginxIp);
                }
            } 
            catch
            {
                // Ignore DNS errors
```